content/posts/2021-09-08-deprecation.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Deprecation of this blog"
+date: 2021-09-08T11:31:15-05:00
+draft: false 
+---
+
+This blog is inactive and will be removed in 2022.
+
+If you are looking for ways to scan R packages for vulnerabilities, you may find the [oysteR package](https://www.jumpingrivers.com/blog/r-package-vulnerabilities-security/) interesting.
+
+For more information on R package security at RStudio, please see [RStudio's support article](https://support.rstudio.com/hc/en-us/articles/360042593974-R-and-R-Package-Security).
+
+TODO: in the event of a malicious package, how would the RSPM team notify customers? How would a package removal be handled?
```

Comment:
I think going through support is the most sane option here, that way we can escalate it to Rick and figure out a plan of action